src/python/main_v.cpp

```diff
@@ -84,6 +84,9 @@ MI_PY_DECLARE(quad);
 // render
 MI_PY_DECLARE(BSDFSample);
 MI_PY_DECLARE(BSDF);
+#if defined(MI_ENABLE_CUDA)
```

Comment:
It isn't necessary to guard those as the `#if def` are already in `optixdenoiser_v.cpp` no?